Explorer/Assets/DCL/Rendering/Reflections/ReflectionProbeRenderer.cs

```diff
@@ -0,0 +1,28 @@
+using UnityEngine;
+
+public class ReflectionProbeRenderer : MonoBehaviour
+{
+    public float IntervalInSeconds = 5f;
```

Comment:
Here we use `[SerializeField] private` instead of `public`

Reply:
done